src/network/providers/ethereum.js

```diff
@@ -155,7 +155,8 @@ const isIdentityAbiRelevant = async () => {
     const abiPath = path.resolve(CONTRACT_BUILD_DIR, 'Identity.json');
 
     try {
-        const abiAndMetadata = JSON.parse(await fs.readFile(abiPath));
+        if (!existsSync(abiPath)) return false;
```

Comment:
Please remove this line. We need to know exactly what happened to the ABI file as long as this part may still contain bugs. Basically if the file doesn't exist, we'll get to the same catch block below where the error will be logged. It's not actually an error, so you may decrease the log level to `warn` instead, but the message itself should remain in the log.